Source/Core/Core/Boot/Boot.cpp

```diff
@@ -282,6 +282,18 @@ static void SetDefaultDisc()
     SetDisc(DiscIO::CreateVolumeFromFilename(config.m_strDefaultISO));
 }
 
+static void CopyDefaultExceptionHandlers()
+{
+  constexpr u32 exception_handler_addresses[] = {0x00000100, 0x00000200, 0x00000300, 0x00000400,
+                                                 0x00000500, 0x00000600, 0x00000700, 0x00000800,
+                                                 0x00000900, 0x00000C00, 0x00000D00, 0x00000F00,
+                                                 0x00001300, 0x00001400, 0x00001700};
+
+  const u32 rti_instruction = 0x4C000064;
```

Comment:
Small nitpick, I'm pretty sure Return From Interrupt is called rfi for PowerPC architecture.

Reply:
Fixed, thanks!